inbac/controller.py

```diff
@@ -152,6 +162,11 @@ def save(self) -> bool:
             self.model.selection_box)
         box: Tuple[int, int, int, int] = self.get_real_box(
             selected_box, self.model.current_image.size, self.model.canvas_image_dimensions)
+        if self.model.selected_fixed_size is not None:
+            box = [box[0],
+                   box[1],
+                   box[0] + self.model.selected_fixed_size[0],
```

Comment:
Fixed size crops ignore the lower right hand corner of the Canvas selection rectangle, since it's approximate due to scaling.